### PR TITLE
Add update rol

### PR DESCRIPTION
**Pull request by moll-y**
**wtf**

### CODE REVIEW

**Comment by andresggz** [comment]
That looks well!

**Comment by andresggz** [comment]
Fix Error and Refactor are right

File: update.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 implements
 other
+refactor
```
Comment (moll-y) [inline update.txt:3]: nt

File: update.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 implements
+other
```
Comment (andresggz) [inline update.txt:2]: Why??

File: update.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 implements
 other
```
Comment (moll-y) [inline update.txt:2]: nt

File: update.txt [hunk 4]
```diff
@@ -1,2 +1,3 @@
 implements
```
Comment (andresggz) [inline update.txt:1]: Please delete this becuse is stupid

File: update.txt (hunk 3 above)
Comment (andresggz) [reply update.txt:1]: Ok, All is right

File: update.txt (hunk 4 above)
Comment (andresggz) [reply update.txt]: Ok! good

**Pull request merged**